### ISSUE TITLE
xlsx-0.7.1 build failure with GHC 8.4

### ISSUE THREAD

**Issue by DanBurton** [qrilka/xlsx#116]
Add semigroup instances
See also: https://ghc.haskell.org/trac/ghc/wiki/Migration/8.4#SemigroupMonoidsuperclasses

```
[ 8 of 39] Compiling Codec.Xlsx.Types.RichText ( src/Codec/Xlsx/Types/RichText.hs, .stack-work/dist/x86_64-osx/Cabal-2.2.0.0/build/Codec/Xlsx/Types/RichText.o )

/Users/dan/scratch/xlsx-0.7.1/src/Codec/Xlsx/Types/RichText.hs:316:10: error:
    • No instance for (Semigroup RunProperties)
        arising from the superclasses of an instance declaration
    • In the instance declaration for ‘Monoid RunProperties’
    |
316 | instance Monoid RunProperties where
    |          ^^^^^^^^^^^^^^^^^^^^
```

I was able to reproduce this locally like so:

```bash
stack unpack xlsx-0.7.1 && cd xlsx-0.7.1
echo 'resolver: nightly-2018-03-21' > stack.yaml
stack build
```

**Comment by qrilka** [comment]
Thanks

**Comment by qrilka** [comment]
Fixed and released as 0.7.2
